Mathlib/LinearAlgebra/Dual/Basis.lean

```diff
@@ -203,6 +203,15 @@ theorem eval_range {ι : Type*} [Finite ι] (b : Basis ι R M) :
     cases nonempty_fintype ι
     rw [← b.toDual_toDual, range_comp, b.toDual_range, Submodule.map_top, toDual_range _]
 
+lemma coord_toDualEquiv_apply [Finite ι] (i : ι) (f : M) :
```

Comment:
Perhaps `dualBasis_coord_toDualEquiv_apply` is a better name for this one?

Reply:
Yeah I agree. Perhaps `@[simp]` too?